server/src/main/java/org/opentosca/toscana/plugins/cloudformation/visitor/PrepareModelNodeVisitor.java

```diff
@@ -31,15 +31,28 @@
     private static final int DEFAULT_WEBAPP_PORT = 80;
 
     /**
-     Create a <tt>PrepareModelNodeVisitor</tt> to prepare a models nodes.
+     Standard constructor.
+     <br>
+     Creates a <tt>PrepareModelNodeVisitor</tt> to prepare a models nodes.
 
-     @param context   TransformationContext to extract topology and logger
-     @param cfnModule module to modify
+     @param context   {@link TransformationContext} to extract the topology and a logger
+     @param cfnModule {@link CloudFormationModule} to modify
      */
     public PrepareModelNodeVisitor(TransformationContext context, CloudFormationModule cfnModule) {
         super(context, cfnModule);
     }
 
+    /**
+     Sets values to default values that are not required by the {@link org.opentosca.toscana.model.EffectiveModel} but by
+     CloudFormation.
+     <br>
+     Passwords need to have a length of 8. If this is not given there is a random password created.
```

Comment:
Maybe rephrase the second sentence to make clear that the random password is used instead of the normal password, like:
`If no password is given, a randomly created password is used instead.`